Content.Server/Explosion/Components/ExplosionResistanceComponent.cs

```diff
@@ -20,11 +18,4 @@ public sealed class ExplosionResistanceComponent : Component
     /// </summary>
     [DataField("damageCoefficient")]
     public float DamageCoefficient = 1;
-
-    /// <summary>
-    ///     Like <see cref="GlobalResistance"/>, but specified specific to each explosion type for more customizability.
-    /// </summary>
-    [ViewVariables(VVAccess.ReadWrite)]
-    [DataField("resistances", customTypeSerializer: typeof(PrototypeIdDictionarySerializer<float, ExplosionPrototype>))]
-    public Dictionary<string, float> Resistances = new();
```

Comment:
Even if its currently unused, IMO this should stay.

Reply:
If its unused it shouldn't stay, as there is no way this will be maintained. It broke bombsuits and nobody noticed for a year.

Reply:
A) Armor has per-damage resistances
B) Add a detailed examine so people can see ingame and it gets spotted a lot sooner than being hidden in yaml, but removing it entirely is not the right call.